docs/letting-go.md

```diff
@@ -0,0 +1,98 @@
+
+## Converting from Data Hub to IML
```

Comment:
This is the page titile? So it should be one header level higher then the others. Currently it's on the same lavel as e.g. "Before you migrate".
Also we are mixing "convert(ing)" and "migrate(ing)". We should stick to one and I prefer migrate.